app/helpers/template_helpers.py

```diff
@@ -250,23 +243,15 @@ def get_static_content_urls(language_code: str, base_url: str, schema_theme: str
         )
 
     elif schema_theme == "census-nisra":
-        help_path = (
-            "help/help-with-the-questions/online-questions-help/"
-            if schema_theme == "census-nisra"
-            else "help/how-to-answer-questions/online-questions-help/"
-        )
+        help_path = "help/how-to-answer-questions/online-questions-help/"
         cookies_path = "cookies/"
         accessibility_statement_path = "accessibility-statement/"
         privacy_and_data_protection_path = "privacy-and-data-protection/"
         terms_and_conditions_path = "terms-and-conditions/"
         contact_us = "contact-us/"
 
     else:
-        help_path = (
-            "help/help-with-the-questions/online-questions-help/"
-            if schema_theme == "census-nisra"
-            else "help/how-to-answer-questions/online-questions-help/"
-        )
+        help_path = "help/help-with-the-questions/online-questions-help/"
```

Comment:
This should be `"help/how-to-answer-questions/online-questions-help/"`?

Reply:
Thanks!